crates/biome_js_analyze/src/lint/style/use_self_closing_elements.rs

```diff
@@ -148,3 +177,15 @@ impl Rule for UseSelfClosingElements {
         ))
     }
 }
+
+#[derive(Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+pub struct UseSelfClosingElementsOptions {
+    pub html: bool,
```

Comment:
The options must be documented. The documentation will be part of the configuration JSON schema, this means that users will be able to see the description when their IDE will autocomplete the option.

Reply:
Ah, I see. I'll add the documentation too

Reply:
Addressed in https://github.com/biomejs/biome/pull/3697/commits/c633de84c42f3c894d0960277be0d216b7bbc459

Please let me know if it's acceptable.